tools/ci_build/build.py

```diff
@@ -1101,8 +1101,8 @@ def run_android_tests(args, source_dir, config, cwd):
         # For Android arm64 abi we are only verify the size of the binary generated by minimal build config
         # Will fail the build if the shared_lib size is larger than the threshold
         if args.minimal_build and config == 'MinSizeRel' and args.build_shared_lib and args.test_binary_size:
-            # set current size limit to 1100KB
-            bin_size_threshold = 1100000
+            # set current size limit to 1170KB
+            bin_size_threshold = 1170000
             bin_actual_size = os.path.getsize(os.path.join(cwd, 'libonnxruntime.so'))
```

Comment:
can we set this to 1155000? That's 100KB larger than the 1.5.2 release size (maybe state that in a comment as well so there's some documentated rationale to the value chosen)

Reply:
I feel that 1165,000 is a better value.

---
In reply to: [510547862](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r510547862) [](ancestors = 510547862)

Reply:
Because? That means we've grown by 150KB, which is ~15%. I'd rather we break well before having to figure out how to reduce 15% worth of increase. 

The 1.5.2 release was 1055616 bytes. 

---
In reply to: [511130614](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r511130614) [](ancestors = 511130614,510547862)

Reply:
1.5.2 release was 1,055,616. so if you want to use 1,165,000 that's 110K larger than 1.5.2 not 101

---
In reply to: [511130614](https://github.com/microsoft/onnxruntime/pull/5540#discussion_r511130614) [](ancestors = 511130614,510547862)